tools/cldr-apps/WebContent/CLDR23Tool.jsp

```diff
@@ -1,318 +1,2 @@
-                                 %>(pathExcludedForSurvey)<%
-                             } else {
-                                 %><b>(pathNOTEXCLUDEDFORSURVEY)</b><%
-                             }
-                         }
-                         
-                         
-                         %><tt style='<%= style %>'><%= x %></tt><br/><%
-                                 
-                     }
-                   %>
-
-
-
-<% } else { %>
-<a href="<%= request.getContextPath() + request.getServletPath() %>">[back]</a>
-<h2><%= l.getDisplayName() %>  [<%= l.getBaseName() %>]</h2>
-<!--  locale -->
-
-    <%
-    
-    CompareCLDRFile ccf = new CompareCLDRFile();
-    
-    CompareCLDRFile.Entry trunk = ccf.add(sm.getDiskFactory().make(l.getBaseName(), false),    "trunk-for-"+sm.getNewVersion());
-    CompareCLDRFile.Entry release = ccf.add(sm.getOldFactory().make(l.getBaseName(), false),   "release-"+ sm.getOldVersion());
-    CompareCLDRFile.Entry vetting = ccf.add(sm.getSTFactory().make(l.getBaseName(), false),    "vetting-"+sm.getNewVersion());
-    
-    %>
-    <br/>
-    
-    
-    <table border=1>
-        <tr>
-            <th>
-                Source
-            </th>
-            <th>
-                Total XPaths
-            </th>
-            <th>
-                Unique XPaths
-            </th>
-         </tr>
-         
-         
-         <%
-         
-         for(CompareCLDRFile.Entry e : ccf.getResults()) {
-        	 %><tr>
-        	           <th><%= e %></th>
-        	           
-                       <td><%= e.getXPaths().size() %></td>
-                       <td><%= e.getUniqueXPaths().size() %></td>
-        	  </tr><%
-         }
-         %>
-         
-            <tr>
-                <th><b>TOTAL</b></th>
-                <td><%= ccf.getAllXPaths().size() %></td>
-                <td>-</td>
-            </tr>
-    </table>
-    
-    <hr>
-    
-    <h3>XPaths in <%= vetting %> but not in <%= trunk %></h3>
-    
-    <%
-    TreeSet<String> vMinusT= new TreeSet<String>(vetting.getXPaths());
-    vMinusT.removeAll(trunk.getXPaths());
-    %>
-
-    <%
-      for(String x : vMinusT) {
-    	  String style = "";
-    	  
-    	  if(m.matches(x, -1)) {
-    		  %>(skip)<%
-    		  style = "background-color: gray; color: white;";
-    		  
-    		  if(rootRes.isPathExcludedForSurvey(x)) {
-    			  %>(pathExcludedForSurvey)<%
-    		  } else {
-    			  %><b>(pathNOTEXCLUDEDFORSURVEY)</b><%
-    			  
-    			  
-    			  PathHeader ph = sm.getSTFactory().getPathHeader(x);
-    			  
-    			  %> ph=<%= ph %> <%
-    		  }
-          }
-    	  
-    	  
-    	  %><tt style='<%= style %>'><%= x %></tt><br/><%
-    			  
-      }
-    %>
-
-<hr>
-<a href="<%= request.getContextPath() + request.getServletPath() %>">[back]</a>
-
-<% } %>
-
-
-</body>
-</html>
\ No newline at end of file
+TODO: is this file still used for anything? If so, document; if not, delete.
```

Comment:
It was used for CLDR 23, specifically.

Reply:
https://github.com/unicode-org/cldr/commit/c59e7dcbf4551ecfabc363542a75cf4f84508068 says 'add test to verify exclude paths in live data'. I think it was probably put in for a specific operational issue.  I'd go ahead and delete the file.